ui/app/components/pricing-metrics-config.js

```diff
@@ -40,35 +41,42 @@ export default Component.extend({
         valueKey: 'defaultReportMonths',
       },
     ];
-  }),
-  modalTitle: computed('model.enabled', function() {
+  }
+
+  @computed('args.model.enabled')
```

Comment:
I didn't know computed was a thing in octane?

Reply:
While the computed decorator is available in octane, I don't think its necessary in this case. I will remove it.